FrameworkSystem/Client/ProxyManagerClient.py

```diff
@@ -610,10 +467,42 @@ def getUserProxiesInfo(self):
 
         :return: S_OK(dict)/S_ERROR()
     """
-    result = RPCClient("Framework/ProxyManager", timeout=120).getUserProxiesInfo()
+    result = RPCClient("Framework/ProxyManager", timeout=120, **self.__extArgs).getUserProxiesInfo()
     if 'rpcStub' in result:
       result.pop('rpcStub')
     return result
 
+  def deleteProxy(self, userDNs):
+    """ Delete proxy
+
+        :param list userDNs: user DNs
+
+        :return: S_OK()/S_ERROR()
+    """
+    return RPCClient("Framework/ProxyManager", timeout=120, **self.__extArgs).deleteProxyBundle(userDNs)
```

Comment:
Do we this function at all? It's just the same as calling `ProxyManagerClient().deleteProxyBundle()`

Reply:
Changed to use `Client` and `createClient` instead `RPCClient`. Function deleted from client.